src/payment/strategies/amazon-pay/amazon-pay-payment-strategy.spec.ts

```diff
@@ -468,7 +470,26 @@ describe('AmazonPayPaymentStrategy', () => {
         }
     });
 
-    describe('When 3ds is enabled', () => {
+    describe( '#initialize', () => {
+        it('throws PaymentMethodNotAllowedError', () => {
+            const remoteCheckout = {
+                referenceId: 'referenceId',
+                billing: {
+                    address: {},
+                    paymentMethodNotAllowed: true,
+                },
+            };
+
+            jest.spyOn(store, 'getState');
+            jest.spyOn(store.getState().remoteCheckout, 'getCheckout')
+                .mockReturnValue(remoteCheckout);
+
+            expect(strategy.initialize({ methodId: paymentMethod.id, amazon: { container: 'wallet' } }))
```

Comment:
As this expect is returning a Promise, please return the last expect in test: 
```suggestion
            return expect(strategy.initialize({ methodId: paymentMethod.id, amazon: { container: 'wallet' } }))
```

Reply:
If add the return the assert fails

Reply:
You'll need to put a `return`. But instead of using `resolves.toThrow`, you'll need to use `rejects.toThrow`: https://jestjs.io/docs/en/expect#rejects